06_puppy/madhavansu/main_test.go

```diff
@@ -0,0 +1,207 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	tassert "github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+/**
+* Map store implemented in below scenarios
+* Dynamic Map ID: MapStore implementation of Storer backed by a map
+* Static Map ID: SyncStore implementation of Storer backed by a sync.Map
+ */
+type storerImpl int
+
+var mapID uint
+
+const (
+	mem storerImpl = iota
+)
+
+var (
+	puppyIN = func() Puppy {
+		return Puppy{
+			id:    107,
+			breed: "Slinky",
+			color: "purple",
+			value: 12999,
+		}
+	}
+	puppyOut1 = func() Puppy {
+		return Puppy{
+			id:    1,
+			breed: "Slinky",
+			color: "purple",
+			value: 12999,
+		}
+	}
+	puppyOut2 = func() Puppy {
+		return Puppy{
+			id:    2,
+			breed: "Slinky",
+			color: "purple",
+			value: 12999,
+		}
+	}
+	modifiedpuppyIN = func() Puppy {
+		return Puppy{
+			id:    107,
+			breed: "Labrador Retriever",
+			color: "brown",
+			value: 13999,
+		}
+	}
+	puppy108 = func() Puppy {
+		return Puppy{
+			id:    108,
+			breed: "Bulldog",
+			color: "red",
+			value: 842.50,
+		}
+	}
+	puppyEmpty = func() Puppy {
+		return Puppy{
```

Comment:
Change this to `return Puppy{}` - let Go fill in the zero values.

Reply:
Updated.